elements/pfe-band/src/pfe-band.scss

```diff
@@ -98,28 +99,28 @@ $variables: (
 // Use-case for transparent backgrounds
 @include pfe-theme-contexts; // imports on="light" etc support
 
-// Pull in pfe-color settings for all supported surface colors
+// Pull in pfe-var settings for all supported surface colors
 @include pfe-surfaces;
 
 // thinner padding on top & bottom
 :host([pfe-size="small"]) {
-  --pfe-band--Padding: calc(#{pfe-local(Padding--vertical)} / 4)  #{pfe-local(Padding--horizontal)};
+  --pfe-#{$LOCAL}--Padding: calc(#{pfe-local(Padding--vertical)} / 4)  #{pfe-local(Padding--horizontal)};
```

Comment:
@castastrophe - I see `#{$LOCAL}` in a bunch of these files instead of using the actual component name. I think this makes reading the code harder and a bit more difficult for debugging issues. If I'm looking the dev tools for and see `--pfe-band--Padding` and then try to find that variable in the repo as I'm doing development, it's going to be hard to find. Do you think this change is necessary?

Reply:
I do - if this is hardcoded and I update the $LOCAL variable at the top, anything using `pfe-local` function will continue to just work but all of these will break.  It might take some work to figure out why all the suddent these variables aren't picking up.

Reply:
I recommend using the temp folder with the CSS assets to debug.

Reply:
@castastrophe - I don't think we'll ever be changing local though. If we did, it'd be a new component, right?

Reply:
Two possible use-cases: You could rename the component variables to add more specificity (nav to navigation or accordion to accordion-header) or you could copy the styles to a new component and want to update it there (without having to grep the entire stylesheet for updates). 🤷

Reply:
I like the idea but the likelihood of those use-cases seems infrequent. Weighing this change against the ease of editing the files from day to day, it seems that we'd want to favor the developer experience being easier. I'd like to err on the side of making our code more readable and approachable but if you'd like to solicit feedback from the larger group, I'd be up for that.

Reply:
I updated the example to use the pfe-print-local function instead. Same idea but it accepts a map instead. Do you prefer that approach? It has the same benefits without hardcoding values.